RELEASE_PROCEDURE.md

```diff
@@ -0,0 +1,18 @@
+# Versioning
+
+Mlibc uses semantic versioning to denote releases.
+A bump in the major version signals an ABI break.
+A bump in the minor version signals a newly released set of features, while still maintaining ABI.
+A bump in the patch signals a bug-fix, while not adding new features.
+
+# Release schedule
+
+A major version will be released irregularly, as it is only necessary when breaking ABI.
+A minor version will be released bi-monthtly (read: every two months).
```

Comment:
It's turn that into: A minor or major version will be released every two months. We can turn the release into a major version if there is a pending ABI break. Additionally, we can promise not to break ABI more than once every 6 months or so.

We should also specify in *which* months we do the release and whether the releases are scheduled at the beginning or end of these months.

Reply:
Fixed.